LondonBritishLibrary/orient/BLorient12513.xml

```diff
@@ -496,88 +593,36 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                        </item>
-                                        <item xml:id="e2">
-                                            <locus target="#IIr"/>
-                                            <desc type="MixedNote">Pen trial</desc>
-                                        </item>
-                                    </list>
-                                </additions>
-                                <bindingDesc>
-                                    <binding xml:id="binding">
-                                        <decoNote xml:id="b1" type="Boards"><material key="leather"/>Wooden boards, repaired in the 
-                                            <placeName ref="INS00001BL">British Museum</placeName>.</decoNote>
-                                        <decoNote xml:id="b2" type="Cover"><material key="leather"/>Partly covered with leather.</decoNote>
-                                    </binding>
-                                </bindingDesc>
-                            </physDesc>
-                            <history>
-                                <origin>
-                                    <origDate notBefore="1600" notAfter="1699">17th century</origDate>    
-                                </origin>
-                                <provenance>This codicological unit was written for a man called 
-                                    <persName ref="PRS14487ZenaGabreel">Zenā Gabrǝʾel</persName>.</provenance>
-                                <acquisition>Presented by the <placeName ref="INS0775LWC">Wellcome Institute</placeName>, 
-                                    <date when="1961-01">January 1961</date>.</acquisition>
-                            </history>
-                    <additional>
-                        <adminInfo>
-                            <recordHist>
-                                <source>
-                                    <listBibl type="catalogue">
-                                        <bibl>
-                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
-                                            <citedRange unit="page">98-99</citedRange>
-                                        </bibl> 
-                                    </listBibl>
-                                </source>
-                            </recordHist>
-                        </adminInfo>
-                    </additional>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">1+147+1</measure>
```

Comment:
Is this correct?
You also have
```
 <measure unit="leaf">1+147+1</measure>
                                        <measure unit="page" type="blank">8</measure><locus target="#Iv #4r #4v #5r #5v #10r #147v #IIv"/>
```
for the entire codex above, then you have part 1 with 3 folia. 
I am not sure that the protective 2 leafs belong to this unit - rather to the entire ms - so should not reappear here. Leaving us with 144 leaves, and 6 blanks, without i and ii

Reply:
There is want endleaf in front and one endleaf in the end, as it is clear from Strelcyns description:
<img width="783" alt="Screenshot 2024_blank leaves BLorient12513 (1)" src="https://github.com/user-attachments/assets/063d15dd-6303-4c05-bd5e-beba0c2652be">
<img width="323" alt="Screenshot 2024_blank leaves BLorient12513" src="https://github.com/user-attachments/assets/fb682587-5acf-47e5-bfb9-2cea3917ac15">
The blank pages concern the verso page of the front endleaf (= Iv) and of the back endleaf (=IIv) and of six pages in the textblock in between.

Reply:
Yes, this is the description for the entire codex. You have it in the main physDesc
My comment referred to the physDesc of msPart p2 where you copypasted the main physDesc

Reply:
OK. I understand. You want me to delete the redundant information in the physDesc of p2.

Reply:
I want the physDesc of p2 to be adjusted to include only information relevant for p2.